CMakeLists.txt

```diff
@@ -163,11 +164,24 @@ if(NOT AWK_EXECUTABLE)
     return()
 endif()
 
-FIND_PACKAGE(LZ4)
-if (NOT LZ4_FOUND)
-    MESSAGE_ONCE(CS_NO_LZ4 "lz4 not found")
-    return()
-endif()
+SET(LINK_WITH_LZ4 FALSE)
+SET(HAVE_LZ4 0)
```

Comment:
you need `CACHE` here too. Because you set the cached value to `TRUE` but never reset it to `FALSE` even if the user decides to recompile without lz4.

Also, I don't really understand why you need *two* flags with exactly the same value, but whatever.

Reply:
fixed.